packages/graphiql-react/src/editor/types.ts

```diff
@@ -3,3 +3,5 @@ import type { Editor } from 'codemirror';
 export type CodeMirrorType = typeof import('codemirror');
 
 export type CodeMirrorEditor = Editor & { options?: any };
+
+export type KeyMapsEnum = 'sublime' | 'emacs' | 'vim';
```

Comment:
Should we just name it `KeyMap`?
- The `Enum` suffix raises the expectation that this is actually an enum and not just a string union type.
- Imho using singular makes more sense as plural because this type represents one possible keymap and not multiple ones at the same time.